pkgs/development/python-modules/pytest-helpers-namespace/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, pytest
+, stdenv
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-helpers-namespace";
+  version = "2019.1.8";
+
+  src = fetchFromGitHub {
+    owner = "saltstack";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "0z9f25d2wpf3lnqzmmnrlvl5b1f7kqwjjf4nzs9x2bpf91s5zny1";
+  };
+
+  propagatedBuildInputs = [ pytest ];
```

Comment:
In case of plugins of pytest we don't actually propagate it. It's an exception to exactly cover situations like you mentioned below.